src/tools/msc.lua

```diff
@@ -368,15 +379,15 @@
 		local result = {}
 
 		for _, enable in ipairs(cfg.enablewarnings) do
-			table.insert(result, '/w1"' .. enable .. '"')
```

Comment:
Why is this being changed?

Reply:
If the goal is to follow VS it may indeed make sense to reject it, although the MSC docs don't mention quotes.
In my case, I have a custom action/generator which somewhere down the line breaks with quotes - granted, that is not an argument to have it changed in the core.

Reply:
Undid those changes.